types/package.json

```diff
@@ -2,11 +2,8 @@
   "name": "upgrade_types",
   "version": "1.0.5",
   "description": "",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
-    "dist/*"
-  ],
+  "main": "src/index.ts",
+  "types": "src/index.ts",
```

Comment:
Just to confirm, types would also be using the same `index.ts` file and not the `index.d.ts` as earlier?

Reply:
Yup now removing the build process from the types and consuming typescript files directly by other projects.

Reply:
okay got it.